src/utilities/effects.ts

```diff
@@ -473,6 +473,68 @@ const effects = {
     }
   },
 
+  async cloneActivityDirectives(
+    activities: ActivityDirective[],
+    plan: Plan,
+    user: User | null,
+  ): Promise<ActivityDirectiveDB[] | undefined> {
+    try {
+      if (plan === null) {
+        throw Error(`Plan is not defined`);
+      }
+      if (!queryPermissions.CREATE_ACTIVITY_DIRECTIVE(user, plan)) {
+        throwPermissionError('cloning activity directives into the plan');
```

Comment:
Prefer "clone" instead of "cloning" here to match other permission error grammar